test/browser.js

```diff
@@ -2,7 +2,7 @@ import test from 'ava';
 import createTestServer from 'create-test-server';
```

Comment:
You can just do `import {serial as test} from 'ava';` to make all of them serial instead of at each callsite.